test/lib/fixtures/projects/unpublish.js

```diff
@@ -0,0 +1,37 @@
+var retrieveTestProjects = require('./test-projects');
+
+var validProjects;
+
+var unpublish = {};
+
+module.exports = function(cb) {
+  if (unpublish.success) {
+    return cb(null, unpublish);
+  }
+
+  retrieveTestProjects(function(err, projects) {
+    if (err) return cb(err);
+
+    validProjects = projects.valid;
+
+    unpublish.success = {
+      default: {
+        url: '/projects/' + validProjects[0].id,
```

Comment:
ah, and here.

Also, keep in mind that this will rely on this test running after the publish test does. Otherwise, you're better off publishing and then unpublishing in the same test.